tools/fiptool/fiptool.c

```diff
@@ -20,8 +20,8 @@
 
 #include <openssl/sha.h>
 
+#include <firmware_image_package.h>
```

Comment:
> Can you move this include to be the alphabetical order, at line 12?

Are you really sure about this?
Other headers exist in users' standard system directories (i.e. outside of ATF project).

We should include public headers first, in the following order:
[1] Standard system headers  (outside of this project)
[2] Headers under the include/ directory of this project
[3] Headers in the local directory (in other words, headers surrounded by "..." instead of angle brackets)

I believe this makes sense because [1] never depends on [2] or [3].
Likewise, [2] never depends on [3]  (if it depends, it is a bug).

Within each category, headers should be sorted alphabetically.
Your suggestion mixes up [1] and [2].  I am opposed to it.

Ideally, I want a blank line between categories.

> could you also put the includes at lines 7,8 and 21 into the right alphabetical order too?

This is a sort within [1], so I can do this.